src/Orckestra.Composer/Providers/Localization/LocalizationExtensions.cs

```diff
@@ -53,34 +54,55 @@ public static string FormatPrice(this ILocalizationProvider localizationProvider
             return string.Format(cultureInfo, format, price);
         }
 
+        private static readonly Dictionary<string, string> CultureInfoToCurrencyCode =
```

Comment:
Not all cultures can be mapped to a RegionInfo, which can cause the ArgumentException depending on the environment. To prevent this, I assume we should consider filtering the cultures by certain criteria or handle possible exceptions when creating RegionInfo objects. For example, the culture da-GL will throw since there is no RegionInfo associated with its LCID

Reply:
// Summary:
    //     Cultures that are specific to a country/region. The names of these cultures follow
    //     RFC 4646 (Windows Vista and later). The format is "<languagecode2>-<country/regioncode2>",
    //     where <languagecode2> is a lowercase two-letter code derived from ISO 639-1 and
    //     <country/regioncode2> is an uppercase two-letter code derived from ISO 3166.
    //     For example, "en-US" for English (United States) is a specific culture.
    SpecificCultures = 2,